src/odemis/gui/cont/tabs.py

```diff
@@ -1899,6 +1922,543 @@ def get_display_priority(cls, main_data):
+        :param cancelled: (bool) if the move is cancelled
+        """
+        stage = self.tab_data_model.main.stage
+        # Get current movement (including unknown and on the path)
+        self.current_position = getCurrentPositionLabel(stage.position.value, stage)
+        self._enable_position_controls(self.current_position, cancelled)
+        # Enable stage advanced controls on milling
+        self._enable_advanced_controls(True) if self.current_position is MILLING else self._enable_advanced_controls(
+            False)
+
+    def _enable_position_controls(self, current_position=None, cancelled=False):
+        """
+        Enable/disable switching position button based on current move
+        """
+        # The move button should turn green only if current move is known and not cancelled
+        if current_position in self.position_btns.keys() and not cancelled:
+            currently_pressed = self.position_btns[current_position]
+            self._toggle_switch_buttons(currently_pressed)
+            currently_pressed.icon_on = img.getBitmap(self.btn_toggle_icons[currently_pressed][1])
+            currently_pressed.Refresh()
+        else:
+            self._toggle_switch_buttons(currently_pressed=None)
+        # Define which button to disable in respect to the current move
+        disable_buttons = {LOADING: MILLING, IMAGING: None, MILLING: COATING, COATING: MILLING, LOADING_PATH: MILLING}
+        for movement, button in self.position_btns.items():
+            if current_position == UNKNOWN:
+                # Only enable loading button when the move is unknown
+                button.Enable() if movement == LOADING else button.Disable()
+            elif movement == disable_buttons[current_position]:
+                button.Disable()
+            else:
+                button.Enable()
+
+    def _enable_advanced_controls(self, enable=True):
+        """
+        Enable/disable stage advanced controls
+        """
+        self.panel.ctrl_milling.Enable(enable)
+        self.panel.stage_align_slider_aligner.Enable(enable)
+        for button in self.btn_aligner_axes.keys():
+            button.Enable(enable)
+
+    def _pause_axis_connectors(self, pause=True):
+        """
+        Pause angle axis connectors (when leaving/entering milling mode)
+        :param pause: (bool) True to pause, False to resume
+        """
+        self.milling_connector.pause() if pause else self.milling_connector.resume()
+
+    def _btn_show_advaned_toggled(self):
+        """
+        Get the value of advanced button for _show_advanced_va_connector ctrl_2_va
+        """
+        return self.panel.btn_switch_advanced.GetValue()
+
+    def _on_show_advanced(self, evt):
+        """
+        Event handler for the Advanced button to show/hide stage advanced panel
+        """
+        self.panel.pnl_advanced_align.Show(self.tab_data_model.show_advaned.value)
+        self.panel.Layout()
```

Comment:
This manages to show the panel albeit the button text is trimmed.
![Screenshot from 2021-01-27 20-37-47](https://user-images.githubusercontent.com/6759564/106044103-f50e5080-60e7-11eb-8233-e72a57a545c9.png)

Reply:
That looks like a known bug in wxWidgets :-( 
I have a workaround for it in gui.cont.tabs.TabBarController._on_tab_change(). However, it needs to be called when the text label is shown. So that's why it doesn't work here out-of-the-box, as it's hidden by default. You'll need to call the function explicitly on the first time the "Advanced" button is pressed.

Reply:
I was meaning to call `_on_tab_change` explicitly but I didn't find a straightforward way to access the `TabBarController` object from the tab. And looking at it I think it'd be better to just make `_fix_big_static_text` usable to both classes. Where it'd be a good place to put it? gui.util? Thanks

Reply:
Let's put it in odemis.gui.util.wx_adapter (it's not a super logical place, but that's where another code to workaround this issue is already situated).
Then, I'd suggest you create a new function with the content of the _on_tab_change():
```python
def fix_static_text_clipping(...):
           _fix_big_static_text(tab.panel)
            # Eventually, update the size of the parent, based on everything inside it
            wx.CallLater(100, _update_layout_big_text, tab.panel)  # Quickly
            wx.CallLater(500, _update_layout_big_text, tab.panel)  # Later, in case the first time was too early
```